cmd/apps/openfaas_app_test.go

```diff
@@ -22,3 +23,77 @@ func Test_getValuesSuffix_aarch64(t *testing.T) {
 		t.Errorf("suffix, want: %s, got: %s", want, got)
 	}
 }
+
+func Test_mergeFlags(t *testing.T) {
+	tests := []struct {
+		name  string
+		flags []string
+		want  map[string]string
+	}{
+		{
+			"openfaas",
+			[]string{
+				"nats.external.enabled=true",
+				"nats.external.host=openfaas-stan.openfaas.svc.cluster.local",
+				"nats.external.port=4222",
+				"nats.clusterName=\"faas-cluster\"",
```

Comment:
Bash may not quote the string in the way you have here, I suspect it has no `\"` character at all.

Reply:
In this case we are responsible for parsing the key/value paris, using plain SH it saddly appears in the go string.

Reply:
You were right, all the double quotes removed in case of sh, bash and fish as well.
I removed the double quote removal from the function with the test cases.